Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.4
+FROM ruby:latest
```

Comment:
Let's stick with a pinned ruby version if that's ok.

Reply:
Sounds good. I'll open up a new issue to deal with desync between the ruby version of the action and the ruby version of the repository that's using the action

Reply:
#3